iele-assemble/README.md

```diff
@@ -1,18 +1,101 @@
 # iele-assemble
 
-## Build instructions
+## For users
 
-### Nix
+### Install - Nix
 
-From the root of the `iele-semantics` repository,
-run `nix-build -A iele-assemble`.
+A Nix expression is provided, with binary caching for Linux and macOS.
```

Comment:
I think the installation instructions should be integrated into the global `INSTALL.md` file (with the caveat under Nix that it will only install the `iele-assemble` program), and these instructions should just link there.